addons/website_event_track_online/views/event_templates.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<!-- Add a shortcut to wishlist / talks after registration -->
+<template id="registration_complete" inherit_id="website_event.registration_complete">
+    <xpath expr="//div[hasclass('row')]" position="after">
+        <div t-if="event.website_track" class="row mt-5">
+            <div class="col-12">
+                <h3>Get prepared ...</h3>
+                <p>Book your seat to the best talks by
+                    <a t-att-href="'/event/%s/track' % (slug(event))">setting up your wishlist</a>
```

Comment:
Wondering if this wishlist feature really has its place in regular "physical" events.
It mostly impacts the notifications and the tracks suggestion, you can't even see your wishlists on a page (or you do?).

But I'm guessing we don't want yet another option on the event for that so meh.

Reply:
Even without the "pure wishlist", you can read talks titles and content and note it somewhere. It's marketing mainly.